model/list.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const listSchema = Schema({
+  type: {type: String, required: true},
+  color: {type: String, required: true},
+  size: {type: String, required: true},
+  timestamp: {type: Date, required: true},
+});
+
+module.exports = mongoose.model('list', listSchema);
```

Comment:
@loganlsr your list schema looks good - just to reiterate, the data types defined in your listSchema do not have to be an object that contains a property of `required: true` - this is simply needed if this field is required to save the associated data - otherwise, you can just apply the data type - ie: `size: String`